sbnanaobj/StandardRecord/SRFlashMatch.cxx

```diff
@@ -11,9 +11,16 @@
 namespace caf
 {
   SRFlashMatch::SRFlashMatch():
-    score(std::numeric_limits<float>::signaling_NaN()),
+    present(false),
     time(std::numeric_limits<float>::signaling_NaN()),
-    pe(std::numeric_limits<float>::signaling_NaN())
+    charge_q(std::numeric_limits<float>::signaling_NaN()),
+    light_pe(std::numeric_limits<float>::signaling_NaN()),
+    score(std::numeric_limits<float>::signaling_NaN()),
+    scr_y(std::numeric_limits<float>::signaling_NaN()),
+    scr_z(std::numeric_limits<float>::signaling_NaN()),
+    scr_rr(std::numeric_limits<float>::signaling_NaN()),
+    scr_ratio(std::numeric_limits<float>::signaling_NaN())
+    // what about chargeXYZ and lightXYZ?
```

Comment:
They are automatically initialized with NaNs

Reply:
You mean that chargeXYZ and lightXYZ get NaNs?
If so then I can just remove the comment.

Reply:
yep, that's defined in the constructor of SRVector3D